File: 1051. Height Checker/heightChecker.js
/**
 * @param {number[]} heights
 * @return {number}
 */
const heightChecker = (heights) => {
  const sortedheights = [...heights].sort((a, b) => a - b);
  let count = 0;

  for (let i = 0; i < heights.length; i++) {
    if (heights[i] !== sortedheights[i]) {
      count++;
    }
  }

  return count;
};


const heights = [1,1,4,2,1,3];
console.log('heightChecker: ', heightChecker(heights));

File: 1051. Height Checker/heightChecker.py
from typing import List

class Solution:
  def heightChecker(self, heights: List[int]) -> int:
    sorted_heights = sorted(heights)
    count = 0
    for i in range(len(heights)):
      if heights[i] != sorted_heights[i]:
        count += 1

    return count


heights = [1,1,4,2,1,3]
solution = Solution()
print('heightChecker: ', solution.heightChecker(heights))
